src/cmsAdapt.js

```diff
@@ -1,23 +1,33 @@
-import { API_BASE } from "../routes/api.js";
-import fetch from "node-fetch";
+import { API_BASE } from '../routes/api.js';
+import fetch from 'node-fetch';
+import { postReview } from '../utils/reviewsUtils.js';
 
 const cmsAdapter = {
   async loadMoviesReviews(id) {
-    const res = await fetch(`${API_BASE}/reviews?filters[movie]=${id}`)
+
+    const res = await fetch(`${API_BASE}/reviews?filters[movie]=${id}`);
+
     const payload = await res.json();
     return payload.data;
   },
   async postReview(review) {
-    const res = await fetch(`${API_BASE}/reviews`, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json'
-      },
-      body: JSON.stringify(review)
-    })
-    console.log(res)
-    return res.json();
-  },
+
+    const author = review.data.author;
+    const rating = review.data.rating;
+
+       if (author && rating) {
```

Comment:
The indentation of this line is off. I would expect it to be aligned with the `const`s above.